package.json

```diff
@@ -5,10 +5,12 @@
   "main": "index.js",
   "dependencies": {},
   "peerDependencies": {
-    "eslint-config-standard": "^4.4.0",
-    "eslint-plugin-no-console-log": "^1.0.0",
-    "eslint-plugin-react": "^3.6.2",
-    "eslint-plugin-standard": "^1.3.1"
+    "babel-eslint": "^5.0.0",
+    "eslint": "^2.1.0",
+    "eslint-config-standard": "^5.1.0",
+    "eslint-config-standard-react": "^2.3.0",
+    "eslint-plugin-react": "^4.2.3",
+    "eslint-plugin-standard": "^1.3.2"
```

Comment:
Remove the `eslint` peer dependency. It makes the config unusable in some cases where it should be usable.